apps/angular/pipe-easy/src/app/indexed.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  standalone: true,
+  name: 'indexed',
+})
+export class IndexedPipe implements PipeTransform {
+  transform(value: string, index: number): string {
+    // very heavy computation
+    return `${value} - ${index}`;
+  }
+}
```

Comment:
you could use a computed signal with a shared function, but pipes are still good for sharing template transformation